en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-graphql-api-in-choreo-connect.md

```diff
@@ -0,0 +1,293 @@
+related to multiple resources. This is not possible with REST APIs. If REST API is used, you will have to consider multiple requests
+to gather the same amount of data. GraphQL APIs are based on a strong type system. Schema Definition Language (SDL) file defines the
+contract between client and the server. It explains how client can access the data provided by the server. This page explains how to 
+deploy a GraphQL API in Choero Connect and also explains the supported features of Choreo Connect for GraphQL APIs.
+
+Choreo Connect supports below features additionally to the basic GraphQL API invocations.
+
+- Runtime configurations
+    - Transport level security
+    - [GraphQL query analysis (query complexity and query depth limitation)]({{base_path}}/design/rate-limiting/graphql-api/overview-query-limits-for-graphql/#static-query-analyzer)
+    - Cross-Origin Resource Sharing (CORS)
+- [Rate limiting policies (API level and operation level) with the support of WSO2 API-M]({{base_path}}/design/create-api/create-a-graphql-api/#rate-limiting-for-graphql-operations)
+- [Security scopes]({{base_path}}/design/create-api/create-a-graphql-api/#authorization-for-graphql-operations)
+- Enable/ disable security for GraphQL operations
+
+!!! note
+    Currently Choreo Connect supports **`query`** and **`mutation`** GraphQL operations only.
+
+You can deploy GraphQL APIs using Choreo Connect by following one of the two deployment modes described in below.
+
+|**Mode**                               | **Method**    |
+|---------------------------------------|---------------|
+|[Choreo Connect with WSO2 API Manager as a Control Plane]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/)   | [Via WSO2 API Manager Publisher Portal](#via-wso2-api-manager-publisher-portal)  |
+|[Choreo Connect as a Standalone Gateway]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway/)                  |[Via apictl for Standalone Mode](#via-apictl-for-standalone-mode)                |
+
+## Via WSO2 API Manager Publisher Portal
+
+Below instructions explain how to deploy a GraphQL to the Choreo Connect via the Publisher Portal in WSO2 API Manager.
+
+!!! info "Before you begin"
+
+    - This guide assumes that you already have a up and running Choreo Connect instance configured with WSO2 API Manager. If not, you can refer to the [Quick Start Guide]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim/) on how to run Choreo Connect with WSO2 API Manager. To learn more about Choreo Connect related componentes, refer to the [Overview of Choreo Connect]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview).
+    - GraphQL schema file (SDL) and a GraphQL server implementaion relevant to the SDL.
```

Comment:
Found https://apim.docs.wso2.com/en/latest/assets/attachments/learn/schema_graphql.graphql in https://apim.docs.wso2.com/en/latest/tutorials/create-and-publish-a-graphql-api/ . Shall we point it here as well? and we can also bring the info about the sample backend in line 126 here.